tools/tpm2_unseal.c

```diff
@@ -38,16 +38,16 @@
 #include <getopt.h>
 #include <sapi/tpm20.h>
 
+#include "../lib/tpm2_options.h"
```

Comment:
Do we really need these relative paths? $(top_srcdir)/lib/tcti is included in the list of directories to search for header files.

Reply:
no we don't, I have no idea how that hunk got added.